### ISSUE TITLE
Improve Cost Breakdown Diagram

### ISSUE THREAD

**Issue by mstich0** [open-plan-tool/gui#251]
We currently have an Overall Cost Breakdown looking like following:
![grafik](https://github.com/open-plan-tool/gui/assets/105202247/eb7b243f-a1b4-4448-b3a0-e07dd43fe7a2)

Unfortunatly the information about the supplier components (electricity/gas/heat) is missing. Since in many approaches the fuel costs will be paid directly at those components, a big part of all costs is missing in this diagram.

**Comment by GlennCeusters** [comment]
I think this is not an "enhancement" but an actual bug. Even the csv export give 0 dispatch / fuel costs.

**Comment by Bachibouzouk** [comment]
Yes we need to fix some bugs in the result processing. This has been delayed because we would like to get as close as possible from the raw `oemof-solph` results. We will sit onto that in the next weeks

**Comment by Bachibouzouk** [comment]
@mstich0 - could you share the project with me?

**Comment by mstich0** [comment]
[fuel_costs.json](https://github.com/user-attachments/files/19530986/fuel_costs.json)
